### PR TITLE
Added new icons for Kubernetes components

### PR DESCRIPTION
**Pull request by nwanduka**
Added icons for Kubernetes components - Token Review and Subject Access Review

**Notes for Reviewers**
I've not updated the Integrations spreadsheet yet. Waiting to get these icons approved first before I go ahead.

For the cytoscape shape, I propose we use a rounded rectangle for both icons.

**[Signed commits](https://github.com/meshery/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by sandramsc — COMMENTED** [review]
LGTM

File: ui/assets/new-icons/Token Review Colored.svg
Comment (ritiksaxena124) [inline ui/assets/new-icons/Token Review Colored.svg:1]: @nwanduka Thank you for this but the lock is used mostly for security or privacy.

File: ui/assets/new-icons/Subject Access Review Colored.svg
Comment (ritiksaxena124) [inline ui/assets/new-icons/Subject Access Review Colored.svg:1]: This looks more like searching for a person. 

These icons don't fit the component you are targeting here but can be used for another component that fits.

**Review by ritiksaxena124 — CHANGES_REQUESTED** [review]

**Comment by leecalcote** [comment]
Title updated.

**Review by goldfishdolphin — APPROVED** [review]
Looking Good

File: ui/assets/new-icons/Subject Access Review Colored.svg
Comment (nwanduka) [reply ui/assets/new-icons/Subject Access Review Colored.svg:1]: Now that you mentioned it, it does look like we're searching for a person. Let me make some adjustments.

File: ui/assets/new-icons/Token Review Colored.svg
Comment (nwanduka) [reply ui/assets/new-icons/Token Review Colored.svg:1]: Oh okay. noted. I'll update the design

**Review by leecalcote — APPROVED** [review]
Will reconcile any changes needed in Figma. 👍 

Thanks, @nwanduka!

**Pull request merged**